### PR TITLE
More developers info : Flake8 and more MyPy

### PR DESCRIPTION
**Pull request by henryiii**
Some additions to dev docs.

### CODE REVIEW

**Comment by henryiii** [comment]
PS: the last force-push added a note on check-manifest, with a tip to keep it from being too slow.

File: pages/developers/style.md [hunk 1]
```diff
@@ -151,7 +168,65 @@ You can also add items to the virtual environment setup for mypy by pre-commit,
...
+`setup.cfg` to enable compatibility with Black:
+
+```ini
+[flake8]
+ignore = E203, E501, W503
+select = C,E,F,W
+```
+
+One recommended plugin for flake8 is `flake8-bugbear`, which catches many
+common bugs.  It is highly opinionated an can be made more so with the `B9`
```
Comment (eduardo-rodrigues) [inline pages/developers/style.md:202]: "...and can …"

**Comment by henryiii** [comment]
> You mean "the one below", I think. While at it, can you then write "Black" for consistency?

Actually, was trying to say the one above *or* Black. I've clarified.

**Comment by eduardo-rodrigues** [comment]
All good. Feel free to merge.

**Comment by henryiii** [comment]
Thanks!

**Pull request merged**